helm-repositories/kaptain/kaptain-repo.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
   name: kaptain-helm-repo
   namespace: kommander-flux
 spec:
-  url: "https://mesosphere.github.io/kaptain-charts"
```

Comment:
This will be updated in the release branch and the release tag will be created, which we will then refer to in `GitRepository` spec in our docs, e.g.:
```
...
ref:
    tag: 2.0.0
  url: https://github.com/mesosphere/kaptain-catalog-applications
```